docs/manuals/core/entities/scenario-cycle-mgt.md

```diff
@@ -98,6 +98,24 @@ propagated to the nested pipelines, tasks, data nodes, and jobs if they are not
 
 If there is more than one scenario in the cycle, an error will be raised if you try to delete a primary scenario. You must promote another scenario as primary before deleting the primary one. On the other hand, if there is only one scenario in the cycle, the primary scenario can be deleted, which will also delete its cycle.
 
+# Compare scenarios
+
+You can compare two or more scenarios, created from the same scenario configuration, by using `taipy.compare_scenarios()`. This function accepts two or more scenarios to be compared, you can also provide the data node config id to specify which data nodes from the scenarios you want to compare.
+
+```python linenums="1"
+import taipy as tp
+from my_config import *
```

Comment:
On my_config.py, there is no comparator defined. Can you create a basic example in the my_config.py with 2 comparators (one for "result_data_node" and one for "date_time_data_node")?

Then we could display the output of both calls to `tp.compare()`.

It would be easier to understand for the user. What do you think ?

Reply:
@jrobinAV I'll change the functions to comparing the `sales_predictions` data node as it's already in the my_config.py. We will create 2 scenario, 1 will have current month set as default datetime(2020, 1, 1), the other will have the current month written as datetime(2020, 2, 1), then we can do the comparison between the previous month scenario and this month scenario on the sales prediction made. What do you think?

Reply:
That sounds great!